docs/design/StratisSoftwareDesign.lyx

```diff
@@ -6933,5 +6933,423 @@ Based on looking at the existing available building blocks, the best option
+
+	
+\begin_inset Quotes eld
+\end_inset
+
+activation_name
+\begin_inset Quotes erd
+\end_inset
+
+: <DEVICEMAPPER_NAME>,
+\end_layout
+
+\begin_layout Plain Layout
+
+	
+\begin_inset Quotes eld
+\end_inset
+
+key_description
+\begin_inset Quotes erd
+\end_inset
+
+: <KEY_DESCRIPTION>,
+\end_layout
+
+\begin_layout Plain Layout
+
+	
+\begin_inset Quotes eld
+\end_inset
+
+pool_uuid
+\begin_inset Quotes erd
+\end_inset
+
+: <POOL_UUID>,
+\end_layout
+
+\begin_layout Plain Layout
+
+	
+\begin_inset Quotes eld
+\end_inset
+
+device_uuid
+\begin_inset Quotes erd
+\end_inset
+
+: <DEVICE_UUID>
+\end_layout
+
+\begin_layout Plain Layout
+
+}
+\end_layout
+
+\end_inset
+
+
+\end_layout
+
+\begin_layout Standard
+See below for an explanation of each JSON value:
+\end_layout
+
+\begin_layout Standard
+\begin_inset listings
+inline false
+status collapsed
+
+\begin_layout Plain Layout
+
+<DEVICEMAPPER_NAME>
+\end_layout
+
+\end_inset
+
+ This is the managed name that will show up in devicemapper when the device
+ is activated using libcryptsetup.
+ While the name can be seen on the system by querying devicemapper, managing
+ encrypted devices outside of Stratis is not supported so this is considered
+ an internal name.
+\end_layout
+
+\begin_layout Standard
+\begin_inset listings
+inline false
+status collapsed
+
+\begin_layout Plain Layout
+
+<KEY_DESCRIPTION>
+\end_layout
+
+\end_inset
+
+
+\end_layout
+
+\begin_layout Standard
+The key description in this token is not used for unlocking, only to refer
+ back to the corresponding LUKS2 keyring token which handles unlocking the
+ device.
+ The key description in both should be identical.
+\end_layout
+
+\begin_layout Standard
+\begin_inset listings
+inline false
+status collapsed
+
+\begin_layout Plain Layout
+
+<POOL_UUID>
+\end_layout
+
+\end_inset
+
+
+\end_layout
+
+\begin_layout Standard
+This UUID corresponds to the Stratis pool that owns this block device.
+ The pool UUID in the token matches the pool UUID in the encrypted Stratis
+ metadata.
+\end_layout
+
+\begin_layout Standard
+\begin_inset listings
+inline false
+status collapsed
+
+\begin_layout Plain Layout
+
+<DEVICE_UUID>
+\end_layout
+
+\end_inset
+
+
+\end_layout
+
+\begin_layout Standard
+This UUID corresponds to the device UUID for this specific block device.
+ The device UUID in the token matches the device UUID in the encrypted Stratis
+ metadata.
+\end_layout
+
+\begin_layout Subsubsection
+Encrypted Stratis metadata format
+\end_layout
+
+\begin_layout Standard
+Once the LUKS2 volume is unlocked, the encrypted Stratis metadata should
+ be accessible through the unencrypted logical devicemapper device exactly
+ the same way as an unencrypted Stratis device.
+\end_layout
+
+\begin_layout Subsection
+Encrypted device creation
+\end_layout
+
+\begin_layout Standard
+Stratis does not wipe the device prior to initializing it with a LUKS2 header.
+ Due to how data is encrypted with LUKS2, if a user is concerned with an
```

Comment:
You might want to reword this paragraph entirely by stating first what encryption of a device with existing data does and does not guarantee. Then, move on and talk about additional actions, beyond handing a device off to Stratis, a user might want to take for further securing previously existing data.